pkg/api/docs/merge/merge_specs.go

```diff
@@ -7,11 +7,19 @@ import (
 	"flag"
 	"fmt"
 	"io/ioutil"
+	"log"
 	"os"
 	"strings"
 
+	"cuelang.org/go/cue/cuecontext"
+	"github.com/getkin/kin-openapi/openapi2conv"
+	"github.com/getkin/kin-openapi/openapi3"
+	"github.com/go-openapi/jsonreference"
 	"github.com/go-openapi/loads"
 	"github.com/go-openapi/spec"
+	"github.com/grafana/grafana/pkg/coremodel/dashboard"
```

Comment:
necessary to support the other change, below

```suggestion
	"github.com/grafana/grafana/pkg/framework/coremodel/registry"
```